### PR TITLE
Add support for NVMe host

### PR DESCRIPTION
**Pull request by P-Cao**
- Add host endpoint
- Add UT for host

### CODE REVIEW

**Comment by P-Cao** [comment]
Test result:
image
Coverage report:
image

File: PyPowerFlex/base_client.py [hunk 1]
```diff
@@ -370,3 +370,17 @@ def _query_selected_statistics(self, action, params=None):
             LOG.error(exc.message)
             raise exc
         return response
+
+    def _modify_entity(self, action, sdc_id, params):
```
Comment (Bhavneet-Sharma) [inline PyPowerFlex/base_client.py:374]: Lets add some documentation for methods.
Same comment applicable for all files.

Comment (Bhavneet-Sharma) [inline PyPowerFlex/base_client.py:374]: I did not find the usage of this method, Where it has been used?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.vscode
+__pycache__
+
+# pytest coverage
+.coverage
+htmlcov
```
[the remark this answers is not in the document]
Comment (P-Cao) [reply .gitignore]: EOF added

File: PyPowerFlex/base_client.py (hunk 1 above)
Comment (P-Cao) [reply PyPowerFlex/base_client.py]: Forgot to remove this method. No need to add extra function in base_client.py. Removed.

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by meenakshidembi691 — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Pull request merged**